### DISCUSSION TITLE
_misc_optm 这个插件会默认udpsize 1200

### DISCUSSION THREAD

**Discussion by ilafz** [IrineSistiana/mosdns#567] [Q&A]
能否手动修改udpsize为4096
bufsize的修改能生效 但是我也想使用_misc_optm里面的其他功能

**Comment by urlesistiana** [comment]
不能。v5 已经去掉修改 udpsize 的功能了。_misc_optm 也去掉了。忘记 udpsize 吧。这已经是 40 年的历史遗留产物了。
